docs/overrides/main.html

```diff
@@ -1,5 +1,10 @@
 {% extends "base.html" %}
 
```

Comment:
This would add an announcement for the entire documentation, we'd want under `batch.md` instead with admonition:

```
???+ warning
    our message...
```

This type of announcement **will** be super useful to let people know we're going to launch v2. So perhaps rephrase to inform them that? I think it'll also need to make it auto-hide so those who already seen don't see it again: https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/?h=announcement#automatic-hiding